Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3028,31 +3028,31 @@ public static InputActionAsset actions
         {
             get
             {
-                if (s_projectWideActions != null)
-                    return s_projectWideActions;
+                if (s_ProjectWideActions != null)
+                    return s_ProjectWideActions;
 
                 #if UNITY_EDITOR
-                s_projectWideActions = Editor.ProjectWideActionsAsset.GetOrCreate();
+                s_ProjectWideActions = ProjectWideActionsAsset.GetOrCreate();
                 #else
-                s_projectWideActions = Resources.FindObjectsOfTypeAll<InputActionAsset>().FirstOrDefault(o => o != null && o.name == kProjectWideActionsAssetName);
+                s_ProjectWideActions = Resources.FindObjectsOfTypeAll<InputActionAsset>().FirstOrDefault(o => o != null && o.name == kProjectWideActionsAssetName);
```

Comment:
Similar as previous comment on FirstOrDefault, should we make sure there are not multiple fulfilling this?